### PR TITLE
Mystory

### PR DESCRIPTION
**Pull request by tipnis**
Please review this, @casey-collab

### CODE REVIEW

File: english/coffee/cookie/bake-cookies.md [hunk 1]
```diff
@@ -5,3 +5,8 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk/walk.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/cookie/bake-cookies.md [hunk 2]
```diff
@@ -5,3 +5,8 @@ You satiate yourself with a pan
...
+[Go for a swim](swim/swim.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: This link is also invalid.

Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/coffee/cookie/Walk/walk.md [hunk 3]
```diff
@@ -0,0 +1 @@
+the walk opens up your mind and lets your creativity flow.
```
Comment (casey-collab) [inline english/coffee/cookie/Walk/walk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/coffee/cookie/Swim/swim.md [hunk 4]
```diff
@@ -0,0 +1 @@
+the swim refresses you and makes you feel fresh and rejuvenated.
```
Comment (casey-collab) [inline english/coffee/cookie/Swim/swim.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tipnis** [comment]
modified again @casey-collab

File: english/coffee/cookie/bake-cookies.md [hunk 5]
```diff
@@ -5,3 +5,9 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk/walk.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/cookie/bake-cookies.md [hunk 6]
```diff
@@ -5,3 +5,9 @@ You satiate yourself with a pan
...
+
+[Go for a swim](swim/swim.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: This link is also invalid.

File: english/coffee/cookie/Walk/walk.md (hunk 3 above)
Comment (casey-collab) [inline english/coffee/cookie/Walk/walk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/coffee/cookie/Swim/swim.md (hunk 4 above)
Comment (casey-collab) [inline english/coffee/cookie/Swim/swim.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tipnis** [comment]
new modify @casey-collab

File: english/coffee/cookie/bake-cookies.md [hunk 7]
```diff
@@ -5,3 +5,9 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk\walk.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/cookie/bake-cookies.md [hunk 8]
```diff
@@ -5,3 +5,9 @@ You satiate yourself with a pan
...
+
+[Go for a swim](swim\swim.md)
```
Comment (casey-collab) [inline english/coffee/cookie/bake-cookies.md]: This link is also invalid.

File: english/coffee/cookie/Walk/walk.md (hunk 3 above)
Comment (casey-collab) [inline english/coffee/cookie/Walk/walk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/coffee/cookie/Swim/swim.md (hunk 4 above)
Comment (casey-collab) [inline english/coffee/cookie/Swim/swim.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/coffee/cookie/bake-cookies.md (hunk 7 above)
Comment (cbuckey-uda) [inline english/coffee/cookie/bake-cookies.md]: You should use a forward slash (`/`), not a backward slash (`\`) in these urls. They are also case sensitive, so make sure you use capital letters if your directories and files have capitals in their names.

**Comment by tipnis** [comment]
finally entered the correct URLs @casey-collab

**Comment by casey-collab** [comment]
Looks great! :palm_tree:

**Pull request merged**